charts/linkerd-crds/templates/gateway.networking.k8s.io_httproutes.yaml

```diff
@@ -11,54 +11,53 @@ metadata:
   labels:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     linkerd.io/control-plane-ns: {{.Release.Namespace}}
-  creationTimestamp: null
```

Comment:
This file is a copy of https://github.com/kubernetes-sigs/gateway-api/blob/main/config/crd/experimental/gateway.networking.k8s.io_httproutes.yaml so we shouldn't remove fields or change formatting.

Reply:
Change reverted